Decsys/Data/Entities/SurveyInstance.cs

```diff
@@ -16,24 +16,22 @@ public class SurveyInstance
         /// Create a SurveyInstance belong to a given Survey.
         /// </summary>
         /// <param name="surveyId">ID of the owning Survey.</param>
+        /// 
+
         public SurveyInstance(int surveyId)
         {
-            Survey = new Survey { Id = surveyId };
+            Survey  = new Survey { Id = surveyId };
         }
 
-        public int Id { get; set; }
+        //Call base constructor for service layer
+        /*public SurveyInstance(int surveyId) : base()
```

Comment:
yeah, we can't use the base constructor as previously discussed, cos this is a LiteDb entity constructing another LiteDb entity 👍 

please remove the commented out ctor we don't need, then 😄